command/schema.go

```diff
@@ -7,44 +7,27 @@ import (
 
 // ResponseSchema is the schema for the data that will be sent out to the client
 type ResponseSchema struct {
-	ID        string                  `jsonapi:"id,primary"`
-	Arguments []schemas.MessagePacket `jsonapi:"arguments,attr"`
-	Count     int                     `jsonapi:"count,attr"`
-	CreatedAt string                  `jsonapi:"createdAt,meta"`
-	Enabled   bool                    `jsonapi:"enabled,attr"`
-	Name      string                  `jsonapi:"name,attr"`
-	Response  EmbeddedResponseSchema  `jsonapi:"response,attr"`
-	Token     string                  `jsonapi:"token,meta"`
+	ID        string          `jsonapi:"primary,id"`
+	Name      string          `jsonapi:"attr,name"`
+	Response  schemas.Message `jsonapi:"attr,response"`
+	Count     int             `jsonapi:"attr,count"`
+	Enabled   bool            `jsonapi:"attr,enabled"`
+	CreatedAt string          `jsonapi:"meta,createdAt"`
+	Token     string          `jsonapi:"meta,token"`
 }
 
 // ClientSchema is the schema the data from the client will be marshalled into
 type ClientSchema struct {
-	Arguments []schemas.MessagePacket `json:"arguments" validate:"required"`
-	Enabled   bool                    `json:"enabled" validate:"required"`
-	Response  EmbeddedResponseSchema  `json:"response" validate:"required"`
+	Context schemas.Context `json:"context" validate:"required"`
+	Enabled bool            `json:"enabled" validate:"required"`
 	// Ignore these fields in user input, they will be filled automatically by the API
-	ID        string `json:"id" validate:"-"`
-	Count     int    `json:"count" validate:"-"`
-	CreatedAt string `json:"createdAt" validate:"-"`
-	Token     string `json:"token" validate:"-"`
-	Name      string `json:"name" validate:"-"`
-}
-
-// EmbeddedResponseSchema is the schema that is stored under the response key in ResponseSchema
-type EmbeddedResponseSchema struct {
-	Action  bool                    `jsonapi:"action,attr" validate:"required"`
-	Message []schemas.MessagePacket `jsonapi:"message,attr" validate:"required,gt=0"`
-	Role    int                     `jsonapi:"role,attr" validate:"gte=0,lte=256"`
-	Target  string                  `jsonapi:"target,attr"`
-	User    string                  `jsonapi:"user,attr"`
+	ID    string `json:"id" validate:"-"`
```

Comment:
None of these fields seem to be automatically populated. (Instead, it looks like they're just "made up" later on.) I'm assuming this is still WIP?